src/Umbraco.Core/PropertyEditors/BlockListConfiguration.cs

```diff
@@ -23,7 +26,7 @@ public class BlockListConfiguration
     public string? MaxPropertyWidth { get; set; }
 
     [DataContract]
-    public class BlockConfiguration
+    public class BlockConfiguration : IBlockConfiguration
```

Comment:
I'm going to go with at "it might be but probably is not" 😆 it "requires judgement" according to MS.

Taken from https://learn.microsoft.com/en-us/dotnet/core/compatibility/

![image](https://user-images.githubusercontent.com/7405322/193816871-5ee919c7-46f4-4576-8268-443f65725b2a.png)